### PR TITLE
Polish help

### PR DESCRIPTION
**Pull request by Duncid**
## Description

Polishing different small things:
- Mostly on Help pannel
- On landing

### CODE REVIEW

**Review by philipperolet — APPROVED** [review]
:+1: modulo linting fix

**Comment by Duncid** [comment]
@philipperolet I'm a bit bothered by the linting (the hasAction thing right?) as I have not touched that.

So I'm not sure about why this error exist and how to fix it.

**Comment by philipperolet** [comment]
will have a quick look & fix then (otherwise we risk breaking the build)

File: front/components/assistant/HelpDrawer.tsx [hunk 1]
```diff
@@ -197,13 +197,14 @@ export function HelpDrawer({
...
-        <div className="flex flex-col gap-2 [&>*]:pl-px">
-          <div className="text-base font-bold">Ask questions</div>
+        <div className="flex flex-col gap-4 [&>*]:pl-px">
+          <Page.SectionHeader title="Ask questions" />
           <AssistantInputBar
             owner={owner}
             onSubmit={handleHelpSubmit}
             conversationId={null}
-            actions={[]}
+            hideAttachment={true}
```
Comment (philipperolet) [inline front/components/assistant/HelpDrawer.tsx:206]: Here, this code was removed in a previous version but your PR reintroduced it

**Comment by philipperolet** [comment]
All good now :+1:

File: front/components/assistant/HelpDrawer.tsx (hunk 1 above)
Comment (Duncid) [reply front/components/assistant/HelpDrawer.tsx]: Thank you !

**Pull request merged**